src/ManifoldMuseum.jl

```diff
@@ -196,12 +308,22 @@ end
 
 Distance such that `log(M, x, y)` is defined for all points within this radius.
 """
-injectivity_radius(M::Manifold, x) = 1.0
+injectivity_radius(M::Manifold, x) = Inf
```

Comment:
This is a duplicate of line 318.

Reply:
No, line 318 differs by an argument. :)

Reply:
Yes, I still seem to have trouble with reading... 😆

Reply:
No worries. :)